src/mono/mono/mini/cpu-s390x.mdesc

```diff
@@ -472,3 +472,171 @@ s390_cij: len:24
 s390_cij_un: src1:i len:24
 s390_cgij: len:24
 s390_cgij_un: len:24
+
+insert_i1: dest:x src1:x src2:i len:12
+insert_i2: dest:x src1:x src2:i len:12
+insert_i4: dest:x src1:x src2:i len:12
+insert_i8: dest:x src1:x src2:i len:12
+insert_r4: dest:x src1:x src2:f clob:1 len:12
+insert_r8: dest:x src1:x src2:f clob:1 len:12
+
+extract_i1: dest:i src1:x len:6
+extract_i2: dest:i src1:x len:6
+extract_i4: dest:i src1:x len:6
+extract_i8: dest:i src1:x len:6
+extract_r4: dest:f src1:x len:12
+extract_r8: dest:f src1:x len:12
+
+xextract_i1: dest:i src1:x src2:i len:6
+xextract_i2: dest:i src1:x src2:i len:6
+xextract_i4: dest:i src1:x src2:i len:6
+xextract_i8: dest:i src1:x src2:i len:6
+xextract_r4: dest:f src1:x src2:i len:10
+xextract_r8: dest:f src1:x src2:i len:10
+
+expand_i1: dest:x src1:i len:12
+expand_i2: dest:x src1:i len:12
+expand_i4: dest:x src1:i len:12
+expand_i8: dest:x src1:i len:12
+expand_r4: dest:x src1:f len:18
+expand_r8: dest:x src1:f len:16
+
+xones: dest:x len:6
+xmove: dest:x src1:x len:6
+xzero: dest:x len:6
+loadx_membase: dest:x src1:b len:26
+storex_membase: dest:b src1:x len:26
+s390_vab: dest:x src1:x src2:x len:6
+s390_vah: dest:x src1:x src2:x len:6
+s390_vaf: dest:x src1:x src2:x len:6
+s390_vag: dest:x src1:x src2:x len:6
+s390_vfasb: dest:x src1:x src2:x len:6
+s390_vfadb: dest:x src1:x src2:x len:6
+s390_vsb: dest:x src1:x src2:x len:6
+s390_vsh: dest:x src1:x src2:x len:6
+s390_vsf: dest:x src1:x src2:x len:6
+s390_vsg: dest:x src1:x src2:x len:6
+s390_vfssb: dest:x src1:x src2:x len:6
+s390_vfsdb: dest:x src1:x src2:x len:6
+s390_vx: dest:x src1:x src2:x len:6
+s390_vo: dest:x src1:x src2:x len:6
+s390_vno: dest:x src1:x src2:x len:6
+s390_vn: dest:x src1:x src2:x len:6
+vector_andnot: dest:x src1:x src2:x len:6
```

Comment:
Just for clarity, it might be preferable to have all the common opcodes before the s390 ones.